components/shared/webrender/display_list.rs

```diff
@@ -119,10 +126,6 @@ impl ScrollTreeNode {
             None => return None,
         };
 
-        if info.scroll_sensitivity != ScrollSensitivity::ScriptAndInputEvents {
-            return None;
-        }
-
```

Comment:
An early return might still be useful here to avoid doing extra work. Just check both axes.